src/helpers/shortkey/suggestion/index.js

```diff
@@ -87,10 +86,14 @@ class Suggestion {
                 return suggestion.phrase.indexOf(mentionText) >= 0;
               }
               this.config.trigger = currentTrigger;
+
+              if (mentionText && mentionText.trim().length <= 0) {
+                return false;
+              }
               return (
                 suggestion.phrase
                   .toLowerCase()
-                  .indexOf(mentionText && mentionText.toLowerCase()) >= 0
+                  .indexOf(mentionText.toLowerCase()) >= 0
```

Comment:
Do we need to check if mentionText exists before we convert it to lowercase?

Reply:
For the safer side now added the mentionText exists before we convert it to lowercase